reference-implementation/lib/transform-stream.js

```diff
@@ -389,18 +389,18 @@ function TransformStreamDefaultSinkCloseAlgorithm(stream) {
   TransformStreamDefaultControllerClearAlgorithms(controller);
 
   // Return a promise that is fulfilled with undefined on success.
-  return flushPromise.then(() => {
+  return PerformPromiseCatch(PerformPromiseThen(flushPromise, () => {
```

Comment:
Looks like you fixed a bug here, and now the `PerformPromiseCatch` is no longer needed.

Reply:
Oh woops, that was actually a typo. The extra `)` on line 403 should have been on line 400 instead.

But you're right, the correct implementation would be to use a single `PerformPromiseThen` call with both handlers. So I *accidentally* fixed a bug. 😅